web/tests/test_forms.py

```
@@ -564,3 +571,258 @@ def test_invalid_profile_form(self):
+            "captcha_1": "PASSED",
+        }
+        # Create a small valid GIF file (1x1 transparent pixel)
+        gif_content = (
+            b"GIF89a\x01\x00\x01\x00\x80\x00\x00\xff\xff\xff\x00\x00\x00!"
+            b"\xf9\x04\x01\x00\x00\x00\x00,\x00\x00\x00\x00\x01\x00\x01\x00\x00\x02\x02D\x01\x00;"
+        )
+        test_image = SimpleUploadedFile(name="test_image.gif", content=gif_content, content_type="image/gif")
+        form_files = {"course_image": test_image}
+
+        form = TeachForm(data=form_data, files=form_files)
+        self.assertTrue(form.is_valid(), msg=form.errors)
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)